BossMod/Autorotation/AutoHints.cs

```diff
@@ -50,10 +52,11 @@ private void OnCastStarted(object? sender, Actor actor)
                 return;
             AOEShape? shape = data.CastType switch
             {
-                2 => new AOEShapeCircle(data.EffectRange), // used for some point-blank aoes and enemy location-targeted - does not add caster hitbox
+                // used for some point-blank aoes and enemy location-targeted - does not add caster hitbox
+                2 => data.EffectRange >= RAIDWIDE_SIZE ? null : new AOEShapeCircle(data.EffectRange),
```

Comment:
null return here will trigger error message later. I'd just check this above and return (eg if EffectRange >= const && data.CastType is 2 or 5)